packages/react-heartwood-components/src/components/Card/components/CardHeader.tsx

```diff
@@ -20,8 +20,15 @@ export interface ICardHeaderProps
 	contextMenu?: IContextMenuProps | null
 }
 
-const CardHeader = (props: ICardHeaderProps): React.ReactElement => {
-	const { title, labelText, labelIcon, actions, contextMenu } = props
+const CardHeader = (
+	props: ICardHeaderProps | IHWCardHeader
+): React.ReactElement => {
+	const propsComp = props as ICardHeaderProps
```

Comment:
Per our convo, let do `reactHeartwoodProps` instead of `propsComp` for `ICardHeaderProps` (the props you've extended on top of IHWCardHeader). propsGql should be `commonProps`.

Goal here is just clarity of what goes where and why without having too much context of the underlying structure